content/docs/rfcs/template/README.md

```diff
@@ -5,7 +5,7 @@ name: (Waku v2 | Logos | Codex) RFC Template
 status: (raw|draft|stable)
 category: (Standards Track|Informational|Best Current Practice)
 tags: an optional list of tags, not standard
-editor: Daniel Kaiser <danielkaiser@status.im>
+editor: Daniel Kaiser <https://github.com/kaiserd>
```

Comment:
OP: https://github.com/vacp2p/rfc/pull/641#discussion_r1422397737

and what I'd propose, for example

```suggestion
editor: Dev Null <devnull@status.im> devnull
```

as

```
name [<email>] [gh handle]
```

kinda like comment on and ssh key

Reply:
@felicio sorry, changed to correct proposal

Reply:
@jimstir you don't have to apologize 😅, it's just something I've just suggested 🙂.

Reply:
others might disagree, of course

Reply:
@felicio you can add other relevant reviewers

Reply:
@jimstir I can't, I don't have the rights on this repo 🙂